src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialServiceSupport.scala

```diff
@@ -59,13 +65,19 @@ trait TrialServiceSupport extends LazyLogging {
 
     // resolve the Thurloe KVPs future
     profileWrappers.map { wrappers =>
+      logger.info(s"makeSpreadsheetValues processing ${wrappers.length} profiles ...")
       // cache a map of subjectId -> ProfileWrapper for efficient lookups later
       val userKVPMap: Map[String, ProfileWrapper] = wrappers.map(pw => pw.userId -> pw).toMap
 
       // loop over all projects (including those that have no defined user) and build spreadsheet rows
       val rows: List[util.List[AnyRef]] = projects.map { trialProject =>
         val rowStrings = trialProject.user match {
-          case Some(user) => getTrialUserInformation(user, userKVPMap).toSpreadsheetValues
+          case Some(user) => Try(getTrialUserInformation(user, userKVPMap).toSpreadsheetValues) match {
+            case Success(s) => s
+            case Failure(x) =>
+              logger.error(s"Failure during getTrialUserInformation for user ${user.userSubjectId} ${user.userEmail} in project ${trialProject.name.value}: ${x.getMessage}", x)
+              errorValues(user.userSubjectId)
+          }
```

Comment:
this is the most important change in the PR. Previously, if we were looping over hundreds of users, a single user who was missing a single key in Thurloe would cause an error that prevented the entire spreadsheet from being updated. Now, with `Try` in place, if a user is missing a key we'll write "ERROR IN FIRECLOUD" into the spreadsheet for that user's row, but keep processing all other users.